nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ReplaceText.java

```diff
@@ -331,6 +332,10 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
             logger.info("Transferred {} to 'failure' due to {}", new Object[] { flowFile, e.toString() });
```

Comment:
If we have multiple exception catches the do the same thing, maybe we shouldn't have separate catches.  It is just repeated code

Reply:
but that is pre-existing to this